src/main.css

```diff
@@ -29,6 +29,16 @@
   font-size: small;
 }
 
+/* large text */
```

Comment:
love the labels.  very smart...probably should do something like that myself :grin: